app/views/submissions/view.js.erb

```diff
@@ -4,7 +4,8 @@ newFile = {
     symbolTree: '<%= j render("submissions/code_symbol_tree", ctag_obj: @ctag_obj) %>',
     url: '<%= url_for(params.permit(:header_position, :action, :controller).to_h) %>',
     versionLinks: '<%= j render("version_links") %>',
-    versionDropdown: '<%= j render("version_dropdown") %>'
+    versionDropdown: '<%= j render("version_dropdown") %>',
+    releaseGrades: '<%= j render("release_grades") %>'
```

Comment:
Should not do any caching on a per-file basis, as the "release/unrelease grades" button applies to the submission as a whole.